tests/Microsoft.Identity.Test.Unit/CoreTests/RegionDiscoveryProviderTests.cs

```diff
@@ -127,7 +127,7 @@ private void AddMockedResponse(HttpResponseMessage responseMessage)
                     new MockHttpMessageHandler
                     {
                         ExpectedMethod = HttpMethod.Get,
```

Comment:
You can also add "ExpectedQueryParams" which would allow you to check the api-version

Reply:
Added